src/features/import-maps.js

```diff
@@ -39,6 +39,9 @@ function processScripts () {
       script.sp = true;
       var fetchPromise = script.src ? fetch(script.src, { integrity: script.integrity }).then(function (res) {
         return res.text();
+      }).catch((err) => {
+        console.error(Error(errMsg(7, process.env.SYSTEM_PRODUCTION ? script.src : 'Failed to fetch ' + script.src)));
```

Comment:
SystemJS Error #7 doesn't look right for this - it is "Failed to fetch module - wrong HTTP status." But in this case we are not fetching a module, and the HTTP status wasn't involved in this.

I think we should maybe introduce a new SystemJS warning for this? Maybe SystemJS W4 (see https://github.com/systemjs/systemjs/blob/master/docs/errors.md for W1, W2, and W3)?

Reply:
Agreed, a new failed to fetch import map warning would be useful here.

Reply:
I've written a new text for W4. Improvements and suggestions are welcome :)